extensions/src/main/scala/pl/touk/nussknacker/extensions/db/pool/DBConnectionPool.scala

```diff
@@ -0,0 +1,8 @@
+package pl.touk.nussknacker.extensions.db.pool
+
+import java.sql.Connection
+
+trait DBConnectionPool {
```

Comment:
we need to close it, otherwise we'll leak connections...

Reply:
Pardon me.. This trait isn't used, I've forgot to remove it.